src/hint.c

```diff
@@ -308,6 +308,7 @@ void ReadClientInfo(ClientNode *np, char alreadyMapped)
    if(np->minWidth == np->maxWidth && np->minHeight == np->maxHeight) {
       np->state.border &= ~BORDER_RESIZE;
       np->state.border &= ~BORDER_MAX;
+      np->state.border &= ~BORDER_MIN;
```

Comment:
About dialog for a fltk application /in my repos/ without this change has close and minimize buttons. On Windows, application compiled without changes that same dialog doesn't have any button, not even close... That is reason for, maybe reasons against are better...